### PR TITLE
add create react app with clerk user management

### LINKED ISSUE #31: Log in with Google
As a user, I want to be able to log in via my Google account so that only I can access data from the sensor.

### PR DESCRIPTION
**Pull request by backevik**
Added Clerk.dev dependency to handle User Management.
- Requires some additional setup for new developers:
  - Create file .env.local in the root of the react-frontend folder
  - Add the following line `REACT_APP_CLERK_FRONTEND_API=A-SECRET-KEY`
  - Ask me for the A-SECRET-KEY
Add Create React App

TODO:
- Trying to figure out how to share Clerk.dev settings.
- Restrict Google Sign in to only Emilie & Andreas emails.

Resolves #31

### CODE REVIEW

File: react-frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (emilieanthony) [inline react-frontend/README.md:1]: We can keep this as it is for now. However, I think we should make sure it is up to date with the project and that the content is relevant to the project further down the road.

File: react-frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Senzr is a tool to track your indoor air quality"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
+    <!--
```
Comment (emilieanthony) [inline react-frontend/public/index.html:13]: Do we need these comments here or can they be seen as redundant? I kind of want to clean up, but maybe the comments are helpful in the future. What do you think?

File: react-frontend/src/App.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+} from '@clerk/clerk-react';
+import Header from './components/Header/Header'
+import Footer from './components/Footer/Footer'
+import './css/styles.css';
+
+function App() {
+  // Retrieve Clerk settings from the environment
+  const clerkFrontendApi = process.env.REACT_APP_CLERK_FRONTEND_API;
+  return (
+    <ClerkProvider frontendApi={clerkFrontendApi}>
```
Comment (emilieanthony) [inline react-frontend/src/App.js:15]: This is really nicely done!

File: react-frontend/src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import './footer.css';
+const Footer = () => {
```
Comment (emilieanthony) [inline react-frontend/src/components/Footer/Footer.jsx:2]: Perfect!

**Review by emilieanthony — APPROVED** [review]
Well done! I really like that React has been introduced to the project.

File: react-frontend/public/index.html (hunk 2 above)
Comment (backevik) [reply react-frontend/public/index.html:13]: Yeah! Go ahead and clean up comments as you wish. Create-react-app generates a lot of stuff that we can probably remove.

**Pull request merged**